Jenkinsfile

```diff
@@ -5,11 +5,11 @@ pipeline {
       steps {
         sh "#!/bin/bash"
```

Comment:
if we don't think there is any need to echo out every command we do, i think we could even put the silencing statement right here, and then make a function that allows you to print certain commands